Raspberry/logFreezerTemp/main.js

```diff
@@ -0,0 +1,63 @@
+const { spawn } = require("child_process");
+const CamOverlayAPI = require("camstreamerlib/CamOverlayAPI");
+const fs = require("fs");
+
+let settings = null;
+try {
+  let data = fs.readFileSync(
+    process.env.PERSISTENT_DATA_PATH + "settings.json"
+  );
+  settings = JSON.parse(data);
+} catch (err) {
+  console.log("No settings file found");
+  return;
+}
+
+let co = new CamOverlayAPI({
+  ip: settings.camera_ip,
+  port: settings.camera_port,
+  auth: settings.camera_user + ":" + settings.camera_pass,
+  serviceID: settings.service_id,
+});
+
+const UNITS = { f: "°F", c: "°C" };
+const RATIOS = { f: [1.8, 32], c: [1, 0] }; //  relation to Celsius
+
+function temperature(num, unit_tag) {
+  r = RATIOS[unit_tag];
+  return (num * r[0] + r[1]).toFixed(2) + " " + UNITS[unit_tag];
+}
+
+async function onePeriod() {
+  let child = spawn(
+    "sudo",
+    [process.env.INSTALL_PATH + "/temper/temper.py", "--json"],
+    {
+      stdio: ["inherit", "pipe", "inherit"],
+    }
+  );
+  child.stdout.on("data", async (data) => {
+    let temp = null;
+    let json_data = JSON.parse(data.toString("utf-8"));
+    if (!json_data[0]) {
+      console.log("No data!");
+    } else {
+      temp = json_data[0]["internal temperature"];
+      console.log(temp);
```

Comment:
pro vsechny logy plati to co uz jsem psal vyse..dal uz je nebudu v review oznacovat

Reply:
asi úplně nechápu proč to chceš smazat? některý jsou zbytečný, ale většina těch logů tam má smysl pro příapdný debug.

Reply:
ja je nechci mazat, chci aby mely vypovidajici hodnotu..tzn. pro cloveka citelna informace..kdyz vylogujes nekde jen cislo tak vis prd jestli je to zapad slunce, tvoje datum narozeni nebo teplota nactena ze senzoru..pak se to zobrazi v SR kamery mezi milionem dalsich logu a musi ti ta informace neco dat

Reply:
přidal jsem tam nějaký návěští teda